React/src/App.css

```diff
@@ -1,4 +1,4 @@
 .main {
-  /* margin: 50px 50px; */
+  margin: 50px;
   width: 90vh;
 }
```

Comment:
Revert change + fix eol

Reply:
I decided to keep this change since this will allow the React sample to have the same look and feel as the jQuery sample.

Regarding eol, I can fix it here, but I think it's better to fix it in the template repo.